core/templates/dev/head/pages/exploration_editor/history_tab/HistoryServices.js

```diff
@@ -192,14 +192,14 @@ oppia.factory('compareVersionsService', [
         if (v1 > v2) {
           throw new Error('Tried to compare v1 > v2.');
         }
-        var explorationDataUrl = (
-          '/createhandler/data/' + explorationData.explorationId + '?v=');
         return $q.all({
-          v1Data: $http.get(explorationDataUrl + v1),
-          v2Data: $http.get(explorationDataUrl + v2)
+          v1Data: EditableExplorationBackendApiService.fetchExploration(
```

Comment:
In the current iteration, EditableExplorationBackendApiService only has fetch. I'll be changing this to pull from the ReadOnlyBackendApiService to use fetch for version types.